elastix_napari/elastix_registration.py

```diff
@@ -85,12 +73,11 @@ def toggle_use_corresponding_points_widget(value):
 
     @widget.advanced.changed.connect
     def toggle_advanced_widget(value):
-        if widget.preset.value == "custom":
-            getattr(widget, "initial_transform").visible = value
-        else:
+        widget.initial_transform.visible = value
```

Comment:
TODO: Maybe `initial_transform` should not be part of the advanced options, given issue #58  🤔